inc/session.class.php

```diff
@@ -114,7 +114,6 @@ static function init(Auth $auth) {
                } else {
                   $_SESSION["glpiauthtype"]     = $auth->user->fields['authtype'];
                }
-               $_SESSION["glpiroot"]            = $CFG_GLPI["root_doc"];
```

Comment:
won't this prevent legacy ui to work on some cases?

Reply:
Right, this line should not be removed.

Reply:
I've re-added it already, it was breaking api tests ;)